src/ert/analysis/_es_update.py

```diff
@@ -683,7 +683,9 @@ def smootherUpdate(
             analysis_config.get_active_module(),
             alpha,
             std_cutoff,
-            global_scaling,
+            analysis_config.get_global_std_scaling()
```

Comment:
As far as I can tell this is not exposed to the user, and is always 1, and since it no longer has a `setter` it might be better to set it explicitly to 1 here (or keep it None and check later)? Then remove the global std scaling from analysis config. Can always reintroduce it later.

Reply:
Seems like it yes, it is gotten from the config dict, but it is never put into the dictionary.

Reply:
Will remove it